scripts/imgtool/image.py

```diff
@@ -254,6 +258,23 @@ def tlv_matches_key_type(tlv_type, key):
 
     return False
 
+def parse_uuid(namespace, value):
+    # Check if UUID is in the RAW format (12345678-1234-5678-1234-567812345678)
+    uuid_re = r'[0-9A-f]{8}-[0-9A-f]{4}-[0-9A-f]{4}-[0-9A-f]{4}-[0-9A-f]{12}'
```

Comment:
dashes should be optional, formatting with them is just pointless overhead in e.g. a cli script

Reply:
Sure, I can add the RAW HEX option to the list.